packages/orchestration/src/orchestration.js

```diff
@@ -112,6 +125,23 @@ const prepareChainAccount = zone =>
         getPort() {
           return this.state.port;
         },
+        /**
+         * @param {Proto3Msg[]} msgs
+         * @param {Omit<TxBody, 'messages'>} [opts]
+         * @throws {Error} if packet fails to send or an error is returned
+         */
+        executeEncodedTx(msgs, opts) {
+          const { connection } = this.state;
+          if (!connection) throw Fail`connection not available`;
+          return E.when(
+            E(connection).send(makeTxPacket(msgs, opts)),
+            // if parsePacketAck cannot find a `result` key, it throws
+            ack => parsePacketAck(ack),
```

Comment:
note that that throw will not be caught by the catch below

Reply:
Correct. This was intended to capture a failure on the initial send. Since we are doing extra handling, I've removed the `catch`